docs/MO_DG/prepare_model/Getting_performance_numbers.md

```diff
@@ -68,22 +65,22 @@ Notice that since the device is CPU, the layers wall clock `realTime` and the `c
 | resnet\_model/add\_5/fq\_input\_1                         | NOT\_RUN   | FakeQuantize | undef                | 0             | 0            |
 
 
-   The `exeStatus` column of the table includes possible values:
-   - `EXECUTED` - layer was executed by standalone primitive,
-   - `NOT_RUN` - layer was not executed by standalone primitive or was fused with another operation and executed in another layer primitive.  
+   The *`exeStatus`* column of the table includes possible values:
+   - The *`EXECUTED`* - layer was executed by standalone primitive.
+   - The *`NOT_RUN`* - layer was not executed by standalone primitive or was fused with another operation and executed in another layer primitive.  
    
-   The `execType` column of the table includes inference primitives with specific suffixes. The layers have the following marks:
-   * Suffix `I8` for layers that had 8-bit data type input and were computed in 8-bit precision
-   * Suffix `FP32` for layers computed in 32-bit precision 
+   The *`execType`* column of the table includes inference primitives with specific suffixes. The layers have the following marks:
+   * Suffix *`I8`* - layers that had 8-bit data type input and were computed in 8-bit precision.
+   * Suffix *`FP32`* - layers computed in 32-bit precision.
 
-   All `Convolution` layers are executed in int8 precision. Rest layers are fused into Convolutions using post operations optimization technique, which is described in [Internal CPU Plugin Optimizations](../../OV_Runtime_UG/supported_plugins/CPU.md).
-   This contains layers name (as seen in IR), layers type and execution statistics.
+   All *`Convolution`* layers are executed in *`int8`* precision. Rest layers are fused into Convolutions using post operations optimization technique (for more details, see the [Internal CPU Plugin Optimizations](../../OV_Runtime_UG/supported_plugins/CPU.md)).
```

Comment:
```suggestion
   All `Convolution` layers are executed in `int8` precision. The rest of the layers are fused into Convolutions using post-operation optimization, as described in [CPU Device](../../OV_Runtime_UG/supported_plugins/CPU.md). 
```

Reply:
I'm not sure why CPU page is referenced here. I didn't find anything related on that page. We need check this in future. @kblaszczak-intel can you check a little bit to see how it's connected?